homeassistant/components/nest/camera_sdm.py

```diff
@@ -251,10 +219,7 @@ async def async_camera_image(
                 return None
             # Nest Web RTC cams only have image previews for events, and not
             # for "now" by design to save batter, and need a placeholder.
-            image = placeholder_image(width=width, height=height)
-            with io.BytesIO() as content:
-                image.save(content, format="JPEG", optimize=True)
-                return content.getvalue()
+            return await self.hass.async_add_executor_job(PLACEHOLDER.read_bytes)
```

Comment:
Does it make sense to load this once and cache it in memory?

Reply:
Sure thing, done.